new-cdc/eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/status/StatusService.java

```diff
@@ -0,0 +1,36 @@
+package io.eventuate.local.common.status;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class StatusService {
+  private volatile CDCStatus status = CDCStatus.CONNECTING;
+  private volatile long processedEvents;
+  private volatile LinkedList<String> lastEvents = new LinkedList<>();
+
+  public CDCStatus getStatus() {
```

Comment:
Instead of individual getters I'd have a single get that returned an object containing all three values.

Reply:
ok